contracts/Constants.sol

```diff
@@ -0,0 +1,19 @@
+pragma solidity ^0.5.0;
+
+contract Constants {
```

Comment:
I would add another function `isZeroBytesLength32(bytes value)` that checks the length of the byte array.